lib/matplotlib/axes/_base.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import (absolute_import, division, print_function,
+﻿from __future__ import (absolute_import, division, print_function,
```

Comment:
There is a Unicode BOM inserted at the beginning of the line here; it should probably be removed.